### PR TITLE
Update Samsung Internet Data, Updating folder mathml/

### PR DESCRIPTION
**Pull request by AdaRoseCannon**
This updated the mathml/ folder part of #5028, every entry is false, because it is not supported.

### CODE REVIEW

**Comment by AdaRoseCannon** [comment]
Travis: "The job exceeded the maximum time limit for jobs, and has been terminated."

File: mathml/elements/maction.json [hunk 1]
```diff
@@ -35,6 +35,9 @@
             "safari_ios": {
```
Comment (edd0408) [inline mathml/elements/maction.json]: [](ur) "samsunginternet_android" {
"version_added". true
}

File: mathml/elements/maction.json [hunk 2]
```diff
@@ -35,6 +35,9 @@
...
               "version_added": false
             },
+            "samsunginternet_android": {
+              "version_added": false
+            },
             "webview_android": {
               "version_added": false
             }
```
Comment (edd0408) [inline mathml/elements/maction.json]: "Version_added" true

File: mathml/elements/maction.json (hunk 1 above)
Comment (edd0408) [reply mathml/elements/maction.json]: Tue

**Comment by edd0408** [comment]
> This updated the mathml/ folder part of #5028, every entry is true because it is n
supported.

**Pull request merged**